packages/shell-api/src/database.spec.ts

```diff
@@ -1796,10 +1796,11 @@ describe('Database', function () {
         },
       });
 
-      const READ_PREFERENCE = {
+      const AGGREGATE_OPTIONS = {
```

Comment:
What's the context for these changes?

Reply:
Since the change just adds a new field to the default constant aggregate options, I just updated the test to match (and the naming since what is being checked is the options as a whole)